### PR TITLE
fix: update dependencies

### PR DESCRIPTION
**Pull request by spaenleh**

### CODE REVIEW

File: src/components/Chatbox/Message.tsx [hunk 1]
```diff
@@ -12,15 +12,14 @@ import { Avatar } from '@graasp/ui';
...
 import {
   DEFAULT_USER_NAME,
   MAX_AVATAR_SIZE,
   MAX_USERNAME_LENGTH,
-} from '@/constants';
-import { useHooksContext } from '@/context/HooksContext';
-import { useChatboxTranslation } from '@/utils/utils';
-
+} from '../../constants';
+import { useHooksContext } from '../../context/HooksContext';
```
Comment (pyphilia) [inline src/components/Chatbox/Message.tsx:21]: You removed the mapper?

File: vite.config.ts [hunk 2]
```diff
@@ -1,15 +1,54 @@
...
+          '@emotion/react',
+          '@emotion/styled',
+          '@mui/material',
+          '@mui/lab',
+          '@mui/icons-material',
+          'react-query',
+          'stylis-plugin-rtl',
+          '@graasp/ui',
+          '@graasp/sdk',
+          '@graasp/translations',
```
Comment (pyphilia) [inline vite.config.ts:50]: sort?

**Review by pyphilia — APPROVED** [review]
🎅

File: package.json [hunk 3]
```diff
@@ -1,137 +1,95 @@
...
+    "@types/react-dom": "^18.2.18",
+    "@types/react-mentions": "^4",
+    "@typescript-eslint/eslint-plugin": "^6.14.0",
+    "@typescript-eslint/parser": "^6.14.0",
+    "@vitejs/plugin-react": "^4.2.1",
+    "eslint": "^8.55.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-react-refresh": "^0.4.5",
+    "path": "^0.12.7",
```
[the remark this answers is not in the document]
Comment (spaenleh) [reply package.json]: I thought, but it did not complain when i removed it, so we are good !

File: src/components/Chatbox/Message.tsx (hunk 1 above)
Comment (spaenleh) [reply src/components/Chatbox/Message.tsx]: I added it back, thank you

**Pull request merged**